src/Http/StrykerDashboardClient.php

```diff
@@ -35,30 +35,25 @@
 
 namespace Infection\Http;
 
-use function curl_close;
-use function is_string;
-use function Safe\curl_exec;
-use function Safe\curl_getinfo;
-use function Safe\curl_init;
-use function Safe\curl_setopt;
+use const PHP_EOL;
+use Psr\Log\LoggerInterface;
 use function Safe\json_encode;
 use function Safe\sprintf;
-use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * @internal
  */
-class BadgeApiClient
+class StrykerDashboardClient
 {
     private const STRYKER_DASHBOARD_API_URL = 'https://dashboard.stryker-mutator.io/api/reports';
```

Comment:
I do prefer having magic constant values as actual constant rather than being inlined: they are easier to find and make it clear that they are acting as a sort of config

Reply:
Using a `private` constant when we could use the literal value instead adds overhead - you need to navigate to the constant to find out about the value.

Reply:
It's an extra overhead to navigate to find out the value, but it's an overhead less to have an expressive name instead of a value to decrypt